packages/internal/src/callback.js

```diff
@@ -94,6 +101,11 @@ harden(makeFunctionCallback);
  * @returns {SyncCallback<I>}
  */
 export const makeSyncMethodCallback = (target, methodName, ...bound) => {
+  isObject(target) ||
+    Fail`sync method callback target must be an object: ${target}`;
+  typeof methodName === 'string' ||
```

Comment:
Can we also allow `number` since that's a valid `PropertyKey`?

Reply:
That's just a typing allowance. Properties are either string or symbol.

Reply:
Hmm, I'd rather have callers convert to string in case of surprises from e.g. `({ [Number.MAX_SAFE_INTEGER + 1]: true })[Number.MAX_SAFE_INTEGER + 2]`.